.vsts.pipelines/steps/setup-macos-native.yml

```
@@ -9,15 +9,44 @@ steps:
     }
     set -x
     # Work around brew bug: https://github.com/dotnet/coreclr/pull/21913
+    # Update to openSSL1.1 due to https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md#openssl
     brew update
     # Install macOS native dependencies not present in hosted pool.
     # Based on https://github.com/sbomer/coreclr/blob/dc2fa1b96415f87d694e5b6e751e38ca6fbd09a1/unix-pipeline.yml#L36-L39
+
     brew install icu4c openssl
+
+    # You might need to "link" pkg-config:
+    brew link pkg-config
```

It doesn't seem quite clear where this comes from. I see it in https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md#openssl, but the nearest above comment that gives me a hint is this, which isn't right:

```
Based on https://github.com/sbomer/coreclr/blob/dc2fa1b96415f87d694e5b6e751e38ca6fbd09a1/unix-pipeline.yml#L36-L39
```

The "you" in "You might need to" is also strange in this context. It's unclear who it's referring to. I think if it were clearer this was copied from a instructions doc it'd be clearer, but I think it would also make sense to not include the comments verbatim.

I think I'd replace this with:

```suggestion
    # "link" pkg-config, related to OpenSSL 1.1: https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md#openssl
    brew link pkg-config
```

And I'd rearrange the steps that really are based on https://github.com/sbomer/coreclr/blob/dc2fa1b96415f87d694e5b6e751e38ca6fbd09a1/unix-pipeline.yml#L36-L39 to be adjacent. (If not possible, I'd add another link comment.)